src/_includes/docs/install/flutter/download.md

```diff
@@ -8,17 +8,17 @@
    {% assign path='C:\\user\\{username}\\dev' %}
    {% assign flutter-path='C:\\user\\{username}\\dev\\flutter' %}
    {% assign terminal='PowerShell' %}
-   {% assign prompt='C:>' %}
+   {% assign prompt='PS C:\\>' %}
```

Comment:
I thought we weren't using :PS C"?

Reply:
We encourage PowerShell everywhere, and the PowerShell prompt is `PS C:\\>`. I noticed it on my PC the other day and wanted to be consistent / correct.